### PR TITLE
cleanup: reduce nested level

### PR DESCRIPTION
**Pull request by zhuangqh**
Signed-off-by: jerryzhuang <jerryzhuang@tencent.com>

#### What type of PR is this?

cleanup

### CODE REVIEW

File: pkg/prediction/dsp/prediction.go [hunk 1]
```diff
@@ -72,7 +72,6 @@ func preProcessTimeSeriesList(tsList []*common.TimeSeries, config *internalConfi
 	for ts := range tsCh {
 		tsList = append(tsList, ts)
 	}
-	wg.Wait()
```
Comment (zhuangqh) [inline pkg/prediction/dsp/prediction.go:75]: duplicate wait here

**Comment by qmhu** [comment]
@kitianFresh Please take a look.

**Comment by kitianFresh** [comment]
Thanks for contributing @zhuangqh

**Pull request merged**